activities.json

```
@@ -252,6 +252,19 @@
     "title": "Contact Picker API",
     "url": "https://wicg.github.io/contact-api/spec/"
   },
+  {
+    "ciuName": null,
+    "description": "This specification extends MediaStreamTrack to let web applications provide an optional media-content hint attribute. This helps sinks like RTCPeerConnection or MediaRecorder select encoder parameters and processing algorithms appropriately based on a hint about the nature of the content being consumed without having to examine the actual content.",
+    "id": "content-hints",
+    "mdnUrl": null,
+    "mozBugUrl": null,
+    "mozPosition": "worth prototyping",
+    "mozPositionDetail": "Content Hints is a welcome higher-level abstraction that does not require broad knowledge and tuning of video encoder, audio-processing, and congestion controls directly. Early concerns over lack of specificity around how hints interact with the lower-level controls they complement have been addressed.",
+    "mozPositionIssue": 101,
+    "org": "W3C",
+    "title": "Content Hints",
```

```suggestion
    "title": "MediaStreamTrack Content Hints",
```

I think it's probably better to use the title that's on the spec here -- it's also much less ambiguous, whereas "Content Hints" could be a lot of different things to somebody familiar with Web technology but not this particular spec.